modules/comment/comment.permissions.js

```diff
@@ -47,8 +47,8 @@ const commentPermissionsMutations = {
     inputDataValidation(REPLY_COMMENT_DATA, replyCommentUpdateValidator),
     hasRoles([ADMIN, SUPERADMIN])
   ),
-  deleteComment: or(isTheSameUser, hasRoles([ADMIN, SUPERADMIN])),
-  deleteReplyForComment: or(isTheSameUser, hasRoles([ADMIN, SUPERADMIN])),
+  deleteComment: or(isTheSameUser, hasRoles([ADMIN, SUPERADMIN, USER])),
+  deleteReplyForComment: or(isTheSameUser, hasRoles([ADMIN, SUPERADMIN, USER])),
```

Comment:
Actually, I don't a problem with the previous code 😃

Reply:
I checked it once more and it is working correctly. If we do not write role USER then the user can't delete his own comment.